package.json

```diff
@@ -71,8 +71,8 @@
 		"nest:start:console:debug": "nest start console --debug --watch --",
 		"nest:test": "npm run nest:test:cov && npm run nest:lint",
 		"nest:test:all": "jest",
-		"nest:test:spec": "jest \"\\.spec\\.ts$\"",
-		"nest:test:e2e": "jest \"\\.e2e-spec\\.ts$\"",
+		"nest:test:unit": "jest \"^((?!\\.api\\.spec\\.ts).)*\\.spec\\.ts$\"",
```

Comment:
Warum ist hier `.api` in den Regex reingekommen?

Reply:
Das ist ein Negative Lookahead, das nur matcht, wenn nicht ".api.spec.ts" im Dateinamen vorkommt. Um nur die Unit-Tests laufen zu lassen, müssen wir ja die API-Tests ausschließen.